src/main/java/be/seeseemelk/mockbukkit/inventory/GrindstoneInventoryMock.java

```diff
@@ -0,0 +1,21 @@
+package be.seeseemelk.mockbukkit.inventory;
+
+import org.bukkit.event.inventory.InventoryType;
+import org.bukkit.inventory.InventoryHolder;
+import org.jetbrains.annotations.NotNull;
+
+public class GrindstoneInventoryMock extends InventoryMock
```

Comment:
You should create another PR implementing a `ResultInventoryMock` we can extend from so it's code doesn't need to be duplicated across multiple inventory mocks. [CraftBukkit source](https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/src/main/java/org/bukkit/craftbukkit/inventory/CraftResultInventory.java).

Reply:
After consideration, we decided not to pursue this